.travis.yml

```diff
@@ -43,8 +44,9 @@ jobs:
       - yarn lint
       - yarn depcheck
       - yarn test
-      - yarn chromatic --quiet --exit-zero-on-changes
-      - yarn start & npx wait-on http://localhost:9001
+      - yarn build-storybook
+      - yarn chromatic --quiet --exit-zero-on-changes --storybook-build-dir docs
+      - yarn http-server docs -p 9001
```

Comment:
```suggestion
      - yarn http-server docs -p 9001 & npx wait-on http://localhost:9001
```

Reply:
You'll need this. `http-server` will never return anything and will hold the Travis executor until Travis times out. `wait-on` will return execution when a ping comes back. You might not need the `wait-on`, but you will need the `&`